pavelib/assets.py

```diff
@@ -128,6 +128,8 @@ def compile_coffeescript(*files):
     ))
 
 
+@task
+@no_help
```

Comment:
So that I can compile SASS without running update_assets, which compiles *all* assets, included those not being changed.